packages/client/package.json

```diff
@@ -23,7 +23,7 @@
     "depcheck": "depcheck --config=.depcheckrc"
   },
   "dependencies": {
-    "@prismicio/types-internal": "^1.5.3",
+    "@prismicio/types-internal": "2.0.0-alpha.7",
```

Comment:
`alpha`? Is there a stable version for the release?

Reply:
Not yet, but it won't change much, if any. @arnaudlewis can explain more about why it is still considered an alpha.

Reply:
I was worried an alpha (eg unstable) might not install on some cases, but seems marked at stable on npm (https://www.npmjs.com/package/@prismicio/types-internal?activeTab=versions) so we should be ok